apis/config/v1beta1/configuration_types.go

```diff
@@ -248,6 +249,13 @@ type RequeuingStrategy struct {
 	// Defaults to null.
 	// +optional
 	BackoffLimitCount *int32 `json:"backoffLimitCount,omitempty"`
+
+	// BaseDelaySeconds defines the base for the exponential backoff for
+	// re-queuing an evicted workload.
+	//
+	// Defaults to 10.
+	// +optional
+	BaseDelaySeconds *int32 `json:"baseDelaySeconds,omitempty"`
```

Comment:
I'm yet wondering about making the exponent and jitter configurable, wdyt @alculquicondor @tenzen-y? I'm

Reply:
I prefer to keep this simple, but @mimowo and/or @alculquicondor have requests from your customers or you have concrete use cases, I'm ok with implementing them.